docs/plugins/ingest-geoip.asciidoc

```diff
@@ -203,3 +203,18 @@ Which returns:
 }
 --------------------------------------------------
 // TESTRESPONSE
+
+[[ingest-geoip-settings]]
+===== Settings
+
+The geoip processor supports the following settings:
+
+`ingest.geoip.cache_enabled`::
+
+    Whether to enable caching of results. Defaults to `false`.
+
+`ingest.geoip.cache_size`::
```

Comment:
We can also simply that and have one single setting actually. If `ingest.geoip.cache_size` = 0, then use a NoCache instance. Otherwise use the one you created.